docs/annotate.rst

```diff
@@ -43,7 +43,11 @@ Step-by-step
 
 #. Connect using the provided credentials
 
-#. Let’s add annotations to Images. We will add Tags to indicate Metaphase stages of these cells.
+#. Let us add annotations to Images.
+
+#. Note that you can add annotations using this interface to any object (or more of them) which you can select in the left-hand side tree or central pane, this means ``Project``, ``Dataset``, ``Image``, ``Screen``, ``Plate`` and ``Well``.
+
+#. First, we will add Tags to indicate Metaphase stages of these cells.
```

Comment:
For " Step-by-step" instructions each step should involve something the user should do. Steps 3 and 4 don't, so maybe combine them with step 5? Or move some of the information to the introduction?

Reply:
fixed in https://github.com/ome/omero-guide-introduction/pull/8/commits/2255c4e0962df4ace84b96ce83fb35a38713876d